### PR TITLE
Create 2024-05-08-TAC.md

### PR DESCRIPTION
**Pull request by maximilien**
Initial minutes from 05/08/2024 meeting.

Please peruse and provide your feedback in comments or inline.

Cc: @Naomi-Wash @planetf1 @ashman-p  and others

### CODE REVIEW

File: meeting-minutes/2024/2024-05-08-TAC.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+[Please check the calendar](https://pqca.org/calendar/)
+
+# Attended by
+
+## TAC members
+
+Norman, Sam, Sophie, and Brian attended
+
+* [ ] Norman Ashley, Cisco
```
Comment (ashman-p) [inline meeting-minutes/2024/2024-05-08-TAC.md]:
```suggestion
* [x] Norman Ashley, Cisco
```

**Review by ashman-p — APPROVED** [review]

**Review by planetf1 — COMMENTED** [review]
LGTM

**Comment by maximilien** [comment]
Thanks @planetf1 but I don’t have a merge button. Not sure why…

**Comment by maximilien** [comment]
Thanks @ryjones

**Pull request merged**